nextstrain/cli/command/build.py

```diff
@@ -98,6 +98,22 @@ def register_parser(subparser):
         dest   = "download",
         action = "store_false")
 
+    # A --logs option doesn't make much sense right now for most of our
+    # runtimes, but I can see how it might in the future.  So we're ready if
+    # that future comes to pass, set up --no-logs as if there's a --logs option
+    # enabled by default.  This also avoids a double negative in conditions,
+    # e.g. avoids writing "if not opts.no_logs".
+    #   -trs, 9 Feb 2023
+    parser.set_defaults(logs = True)
+    parser.add_argument(
+        "--no-logs",
+        help   = "Do not show the log messages of the remote build. "
+                 "Currently only supported when also using --aws-batch. "
+                 "Default is to show all log messages, even when attaching to a completed build."
+                  f"{SKIP_AUTO_DEFAULT_IN_HELP}",
+        dest   = "logs",
+        action = "store_false")
```

Comment:
I'm confused by the use of `set_defaults` here. I thought using `store_false` will already set the default value to `True`?

Reply:
It's btw not the PR that's causing tests to fail - CI fails also on master: https://github.com/nextstrain/cli/actions/runs/4266385449

I've made a PR to bump minimum supported Python version to 3.7: #256

Reply:
Ah, it will, but I found it clearer to be explicit given that the destination for `--no-logs` is `logs`. If you think it's much more confusing, I can remove the `set_defaults` call.

Reply:
Ah, no need to remove it. Just wanted to make sure I wasn't missing something.